cli/mender-artifact/main.go

```diff
@@ -162,6 +162,12 @@ func getCliContext() *cli.App {
 			Usage: "Full path to the state script(s). You can specify multiple " +
 				"scripts providing this parameter multiple times.",
 		},
+		cli.BoolFlag{
+			Name: "no-provides",
```

Comment:
> The flag is added, however, what are your thoughts on it guiding this one addition, or should it also guard agains the user having added provides flags themselves?

Hmm, good question. I think I'm leaning towards not disabling manually added ones, this seems like overreaching. IOW leave it as is.

However, it does bring into question the name of the flag, which is then a bit misleading. May I suggest renaming it to `no-checksum-provide`, to be more explicit?